starboard/extension/setstring_test.cc

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is a kind of death test, as described in
+// https://github.com/google/googletest/blob/main/docs/advanced.md#death-tests,
+// except, it doesn't spawn another process, but terminates itself.
+// It is intended to be started from a host machine on a target device
+// to generate a coredump, which is verified on the host.
+
+#include "starboard/client_porting/wrap_main/wrap_main.h"
+#include "starboard/event.h"
+#include "starboard/extension/crash_handler.h"
+#include "starboard/system.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+const char* long_string =
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "LongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLong"
+    "String";
+
+TEST(ExtensionTest, SetString) {
+  typedef CobaltExtensionCrashHandlerApi ExtensionApi;
+  const char* kExtensionName = kCobaltExtensionCrashHandlerName;
```

Comment:
Is this variable needed? It seems more clear to just pass `kCobaltExtensionCrashHandlerName` directly to `SbSystemGetExtension()`.

Reply:
Fixed.